src/content/1.7/development/architecture/file-structure/_index.md

```diff
@@ -0,0 +1,252 @@
+
+### /controllers
+
+Contains controllers – as in Model-View-Controller (or MVC), the software architecture used by PrestaShop. Each file controls a specific part of PrestaShop.
+
+Controllers are separated in two directories:
+
+- `/admin`: contains legacy controllers for the Back Office
+- `/front`:  contains controllers for the Front Office
+
+### /docs
+
+Contains documentation files, licenses, sample import files, and more. 
+
+{{% notice tip %}}
+You should delete this directory in a production environment.
+{{% /notice %}}
+
+### /download
+
+Contains your virtual products, which can be downloaded by the customers who bought them. Files are stored with an md5 filename.
+
+### /img
+
+Contains all of PrestaShop's default images, icons and picture files – that is, those that do not belong to the theme. 
+
+It contains the following subdirectories:
+
+- `/admin`: Back office images 
+- `/c`: Category pictures
+- `/cms`: CMS pictures
+- `/co`: Attributes (colors) pictures
+- `/flags`: Country & language flags {{< minver v="1.7.7" >}}
+- `/genders`: Gender (_Shop Parameters > Customers > Client titles_) pictures
+- `/jquery-ui`: jQuery UI images
+- `/l`: Language pictures
+- `/m`: Manufacturer pictures
+- `/os`: Order state pictures
+- `/p`: Product pictures
+- `/s`: Carrier pictures
+- `/scenes`: Category scene pictures (removed since 1.6)
+- `/st`: Stores (_Shop Parameters > Contact > Stores_) pictures
+- `/su`: Supplier pictures
+- `/t`: Tab icons
+- `/tmp`: Temporary pictures
+
+### /install
+
+Contains all the files related to PrestaShop's installer. This directory is named `install-dev` when using PrestaShop from source code.
+
+{{% notice note %}}
+For security reasons, you will be required to delete this directory after installation.
+{{% /notice %}}
+
+### /js
+
+Contains JavaScript files that are not attached to themes. Most of them belong to the back office. 
+
+This is also where you will find the jQuery framework.
+
+### /localization
+
+Contains all of PrestaShop's localization packs — that is, files that contain local information such as default currencies and languages, tax rules and tax rule groups, states and measurement units in use in each country.
+
+The [CLDR database](http://cldr.unicode.org/) is also located in this directory.
+
+### /mails
+
+Contains all HTML and text templates for e-mails sent by PrestaShop. Each language has its specific folder, where you can manually edit their content if you wish. PrestaShop contains a tool to edit your e-mails, located in the back office, in the Localization &gt; Translation page.
+
+Since {{< minver v="1.7.6" >}} this directory also contains [Mail themes]({{< ref "/1.7/development/components/mail-templates/_index.md" >}}).
+
+### /modules
+
+Contains all of PrestaShop's modules, each in its own folder. 
+
+{{% notice tip %}}
+If you wish to definitely remove a module, first uninstall it from the back office.
```

Comment:
```suggestion
If you wish to definitely remove a module, it is better to trigger the uninstall process from the back office before deleting the module folder
```

Reply:
If I'm not mistaken when you uninstall a module from the back office you're given the option to keep or to remove the module's files.

```suggestion
If you wish to definitely remove a module, it is better to uninstall it from the back office.
```

Wdyt?